nservicebus/upgrades/support-policy.md

```diff
@@ -77,9 +77,9 @@ Therefore the recommended approach is to upgrade *one* major version at a time,
 
 ## FAQ
 
-**Does this policy apply to the [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/), [ServicePulse](/servicepulse/) applications, or [Previews](/previews/)?**
+**Does this policy apply to the [ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/), [ServicePulse](/servicepulse/) applications, or [previews](/previews/)?**
 
-No, the policy applies only to the supported NServiceBus packages listed on the [NServiceBus Packages Supported Versions](supported-versions.md) page. The ServiceControl support policy is outlined [here](/servicecontrol/upgrades/support-policy.md). The support policy for Previews is outlined [here](/previews/support-policy.md).
+No, the policy applies only to the supported NServiceBus packages listed on the [NServiceBus Packages Supported Versions](supported-versions.md) page. The ServiceControl support policy is outlined [here](/servicecontrol/upgrades/support-policy.md). The support policy for previews is available [here](/previews/support-policy.md).
```

Comment:
Linking "here" is bad for screen readers and search engines alike.

```suggestion
No, the policy applies only to the supported NServiceBus packages listed on the [NServiceBus Packages Supported Versions](supported-versions.md) page. The ServiceControl support policy is outlined [here](/servicecontrol/upgrades/support-policy.md). See the [support policy for previews](/previews/support-policy.md) for more information.
```

Reply:
I agree. I've always been averse to it, but it seems like we do it more and more and I heard arguments that it's now acceptable. I'll add it back to my list of things to abhor. 😉